test/unit/component/parser/json/BpmnJsonParser.callActivity.test.ts

```diff
@@ -34,6 +34,8 @@ describe('parse bpmn as json for callActivity', () => {
         name: `call activity name`,
         calledElement: 'process_1',
         isExpanded: isExpanded,
+        incoming: 'flow_in',
```

Comment:
IMHO, I don't think we should use the attributes `incoming` & `outgoing` of the `bpmnElement` of a shape.
Because, not all vendors implement these attributes in their generated BPMN files.
The way the most safe is to use the data from the `sequenceFlow`, `messageFlow`, `annotation`...

Reply:
Good catch. 
As discussed together, we are going to keep the usage of the `incoming` & `outgoing` of the `bpmnElement`. This lets implement other features that require them to enrich data returned by the public API (#2503 for instance).
We will improve the parsing code later by enriching the arrays with values taken from the `sourceRef` and `targetRef` from flow elements (issue to be created).